styleguide/source/_patterns/01-molecules/06-components/13-subcategory-topics-description/subcategory-topics-description.twig

```diff
@@ -0,0 +1,35 @@
+{#
+ # @file subcategory-topics-description.twig
+ # Copyright 2017 Palantir.net, Inc.
+ #}
+
+<div class="subcategory-topics-description layout">
+  <div class="subcategory-topics-description_wrap">
+    <div class="subcategory-topics-description_title">
+      <div class="subcategory-topics-description_title-inner">
+        {% include 'atoms-eyebrow' %}
+        <a href="{{ url }}" class="subcategory-topics-description_link">Get Started</a>
+      </div>
+    </div>
+    <div class="subcategory-topics-description_teaser">
+      {% include 'atoms-paragraph' with { 'content': 'Nulla vitae elit libero, a pharetra augue. Donec id elit non mi a porta gravida at eget metus. Cras justo odio, dapibus ac facilisis in, egestas eget quam.', 'class': 'type-smaller category-highlight_teaser' } %}
```

Comment:
I don't think this needs `type-smaller`. On both http://ama-assn.local/education and the designs it looks like it's the same size as the link text. Please remove `type-smaller`.